app/Resources/views/inside-elife-article.html.twig

```diff
@@ -22,16 +22,14 @@
 
         {% block main %}
 
-            {% for block in blocks %}
-                {{ render_pattern(block) }}
-            {% endfor %}
-
             {% if hypothesisOpener is defined %}
-
                 {{ render_pattern(hypothesisOpener) }}
-
             {% endif %}
 
+            {% for block in blocks %}
```

Comment:
I'd like to consider the `hypothesisOpener` as part of the list of blocks, so it can be excluded from the Twig templates entirely.